proto/telemetry.proto

```diff
@@ -19,6 +19,17 @@ enum TelemetryNodeType {
   TELEMETRY_NODE_TYPE_COMPACTOR = 4;
 }
 
+// Coo
+enum TelemetryClusterType {
+  TELEMETRY_CLUSTER_TYPE_UNSPECIFIED = 0;
+  TELEMETRY_CLUSTER_TYPE_SINGLE_NODE = 1;
+  TELEMETRY_CLUSTER_TYPE_TEST = 2;
+  TELEMETRY_CLUSTER_TYPE_DOCKER_COMPOSE = 3;
+  TELEMETRY_CLUSTER_TYPE_KUBERNETES = 4;
+  TELEMETRY_CLUSTER_TYPE_HOSTED = 5;
+  TELEMETRY_CLUSTER_TYPE_STANDALONE = 6;
```

Comment:
These concepts are not orthogonal. For example, the CI/CD pipeline may run tests in standalone mode, making it confusing to different people.

I tend to use 2 separated dimensions

1. Is it a test or development instance? If yes, should be excluded from telemetry. I think our existing `TELEMETRY_ENABLED` is enough for this usage.
2. If it's a production cluster, how it is deployed? e.g. docker-compose, Kubernetes or standalone.

Reply:
Make sense, let's make the `TELEMETRY_ENABLED` default to true.